src/lib/slayer/nieveTasks.ts

```diff
@@ -0,0 +1,530 @@
+import { Monsters } from 'oldschooljs';
+
+const nieveTasks = [
+	{
+		name: 'Aberrant spectre',
+		amount: [120, 185],
+		extendedAmount: [200, 250],
+		requirements: {
+			slayerLevel: 60,
+			combatLevel: 65
+		},
+		alternatives: ['Deviant spectres'],
+		weight: 6,
+		ID: Monsters.AberrantSpectre.id
+	},
+	{
+		name: 'Abyssal demon',
+		amount: [120, 185],
+		extendedAmount: [200, 250],
+		requirements: {
+			slayerLevel: 85,
+			combatLevel: 85
+		},
+		alternatives: ['Abyssal sire', 'Greater abyssal demon'],
+		weight: 9,
+		ID: Monsters.AbyssalDemon.id
+	},
+	{
+		name: 'Adamant dragon',
+		amount: [3, 7],
+		extendedAmount: [20, 30],
+		requirements: {
+			questPoints: 35
+		},
+		alternatives: ['Adamant dragon'],
+		weight: 2,
+		ID: Monsters.AdamantDragon.id
+	},
+	{
+		name: 'Ankou',
+		amount: [50, 90],
+		extendedAmount: [90, 150],
+		requirements: {
+			combatLevel: 40
+		},
+		alternatives: ['Dark Ankou'],
+		weight: 5,
+		ID: Monsters.Ankou.id
+	},
+	{
+		name: 'Aviansie',
+		amount: [120, 185],
+		extendedAmount: [130, 250],
+		requirements: {
+			unlocked: [10]
+		},
+		alternatives: [
+			"Kree'arra",
+			'Flight Kilisa',
+			'Flockleader Geerin',
+			'Wingman Skree',
+			'Reanimated aviansie'
+		],
+		weight: 6,
+		ID: Monsters.Aviansie.id
+	},
+	{
+		name: 'Basilisk',
+		amount: [120, 180],
+		extendedAmount: [200, 250],
+		requirements: {
+			slayerLevel: 40,
+			unlocked: [25]
+		},
+		alternatives: ['Basilisk Knight'],
+		weight: 6,
+		ID: Monsters.Basilisk.id
+	},
+	{
+		name: 'Black demon',
+		amount: [120, 185],
+		extendedAmount: [200, 250],
+		requirements: {
+			combatLevel: 80
+		},
+		alternatives: ['Demonic gorilla', 'Balfrug Kreeyath', 'Porazdir', 'Skotizo'],
+		weight: 9,
+		ID: Monsters.BlackDemon.id
+	},
+	{
+		name: 'Black dragon',
+		amount: [10, 20],
+		extendedAmount: [40, 60],
+		requirements: {
+			combatLevel: 80,
+			questPoints: 32
+		},
+		alternatives: ['Baby black dragon', 'King Black Dragon', 'Brutal black dragon'],
+		weight: 6,
+		ID: Monsters.BlackDragon.id
+	},
+	{
+		name: 'Bloodveld',
+		amount: [120, 185],
+		extendedAmount: [200, 250],
+		requirements: {
+			combatLevel: 50,
+			slayerLevel: 50
+		},
+		alternatives: ['Mutated Bloodveld'],
+		weight: 9,
+		ID: Monsters.Bloodveld.id
+	},
+	{
+		name: 'Blue Dragon',
+		amount: [120, 185],
+		extendedAmount: null,
+		requirements: {
+			combatLevel: 65,
+			questPoints: 32
+		},
+		alternatives: ['Blue dragon', 'Baby blue dragon', 'Brutal blue dragon', 'Vorkath'],
+		weight: 4,
+		ID: Monsters.BlueDragon.id
+	},
+	{
+		name: 'Boss',
+		amount: [3, 20],
```

Comment:
```suggestion
		amount: [3, 35],
```
special cases for barrows of 36 and zulrah of 15

Reply:
fixed in src/lib/slayer/bossTasks -- special case accounted for in the slayertask command